src/guide/code-owners.md

```diff
@@ -54,57 +57,91 @@ This list is intended to make it easier to identify which email list to include
     * `logging` – Runtime
     * `memory` – Runtime, GC
     * `metaprogramming` – Runtime
```

Comment:
metaprogramming should probably be HotSpot instead of Runtime. The rest of the HotSpot directories look fine to me.